applications/CoSimulationApplication/tests/test_CoSimulationApplication.py

```diff
@@ -63,6 +64,7 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestPingPong]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestConvergenceAcceleratorWrapper]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestTinyFetiCoSimulationCases]))
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestThermalRomCoSim]))
```

Comment:
how long does it run?

Reply:
On my computer, which is a bit slow, it takes approximately 0.2-0.3 seconds to execute.In the CI environment, the execution time is as follows: 
"testConvDiffStationaryRom2DCoSim (test_thermal_rom_co_sim.TestThermalRomCoSim) ... 0.208s ...ok".

Reply:
In the CI environment, the execution time is as follows: 
"testConvDiffStationaryRom2DCoSim (test_thermal_rom_co_sim.TestThermalRomCoSim) ... 0.208s ...ok".

Reply:
Now, it is doing 4 iterations. I could reduce it to 2 iterations only. What do you think?

Reply:
I think thats ok
if you can reduce it further then great, otherwise I am fine as is

Reply:
I reduced it.